build/dashboards/kubernetes-monitoring.json

```diff
@@ -390,7 +390,7 @@
           "tableColumn": "",
           "targets": [
             {
-              "expr": "sum (container_fs_usage_bytes{device=~\"^/dev/((xv|s|h)d[a-z][0-9]|nvme[0-2].*)\",id=\"/\",kubernetes_io_hostname=~\"^$Node$\"}) / sum (container_fs_limit_bytes{device=~\"^/dev/((xv|s|h)d[a-z][0-9]|nvme[0-2].*)\",id=\"/\",kubernetes_io_hostname=~\"^$Node$\"}) * 100",
+              "expr": "sum (container_fs_usage_bytes{device=~\"^/dev/((xv|s|h|v)d[a-z][0-9]|nvme[0-2].*)\",id=\"/\",kubernetes_io_hostname=~\"^$Node$\"}) / sum (container_fs_limit_bytes{device=~\"^/dev/((xv|s|h|v)d[a-z][0-9]|nvme[0-2].*)\",id=\"/\",kubernetes_io_hostname=~\"^$Node$\"}) * 100",
```

Comment:
the older regexp allowed string like 
`/dev/hda`, whereas the new regexpr won't allow, as it requires 1 digit. what about examples like:
`/dev/hda` but also `/dev/hda11`
perhaps add `.*` also after `[0-9]` ?

Reply:
This is a good question.
In OCP of fyre, ibmcloud aws, none of them have more than 4 disks/partitions. This query should work.
But to support GKE, this is not enough which got 1 disk and 12 partitions.
So I think this advice is good.